R/Achilles.R

```diff
@@ -631,6 +642,18 @@ achilles <- function (connectionDetails,
                                 achillesTables = unique(achillesTables))
   }
   achillesSql <- c(achillesSql, indicesSql)
+
```

Comment:
Can you add a section comment like this:

`# Optimize Atlas Cache -----------------------------------------`

Reply:
fixed